pkg/util/utils.go

```diff
@@ -344,6 +346,10 @@ func BuildBasicTags(cluster string, obj interface{}, namespaceID types.UID) []mo
 			Scope: String(common.TagScopeCluster),
 			Tag:   String(cluster),
 		},
+		{
+			Scope: String(common.TagScopeVersion),
+			Tag:   String(common.TagValueVersion10),
```

Comment:
where is TagValueVersion10, can we set value with major/minor/patch version like 1.0.0

Reply:
TagValueVersion10 is 1.0 defined in 
https://github.com/vmware-tanzu/nsx-operator/blob/vpc_dev/pkg/nsx/services/common/types.go#L64
yes, we can set 1 as major version, .x as minor version, the last .x as patch version.